presto-main/src/main/java/com/facebook/presto/execution/CreateFunctionTask.java

```diff
@@ -66,7 +70,12 @@ public String explain(CreateFunction statement, List<Expression> parameters)
     public ListenableFuture<?> execute(CreateFunction statement, TransactionManager transactionManager, Metadata metadata, AccessControl accessControl, QueryStateMachine stateMachine, List<Expression> parameters)
     {
         Analyzer analyzer = new Analyzer(stateMachine.getSession(), metadata, sqlParser, accessControl, Optional.empty(), parameters, stateMachine.getWarningCollector());
-        analyzer.analyze(statement);
+        Analysis analysis = analyzer.analyze(statement);
+        if (analysis.getFunctionHandles().values().stream()
+                .anyMatch(SqlFunctionHandle.class::isInstance)) {
+            throw new PrestoException(NOT_SUPPORTED, "Invoking a SQL function in SQL function body is not yet supported");
```

Comment:
Just say "not supported". The "yet" makes it sound like something coming in the near future. 😂

Reply:
I think it's probably better to say "Invoking a dynamically registered function in SQL function body is not supported".